services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/tormaystarkastelu/TormaystarkasteluLaskentaServiceTest.kt

```diff
@@ -50,16 +73,372 @@ internal class TormaystarkasteluLaskentaServiceTest {
         assertThat(perusindeksi).isEqualTo(indeksi)
     }
 
+    @Nested
+    inner class KatuluokkaLuokittelu {
+        // The parameter is only used to call mocks
+        val geometriat = listOf<Geometriat>()
+
+        @Nested
+        inner class WithYlreParts {
```

Comment:
I don't know what ylre means. Is that a term that a haitaton developer should know, or should that be renamed?

Reply:
I think YLRE is Yleisten alueiden rekisteri: https://hri.fi/data/fi/dataset/helsingin-kaupungin-yleisten-alueiden-rekisteri.

I used the database table names in some places because the Finnish ones didn't offer much more context. But they were harder to distinguish from one another:
- Yleinen katuosa
- Yleinenkatualue katuluokka (Or just Yleinen katualue in one comment)
- Liikenteellinen katuluokka

These all just sound like the same thing to me.

I think I don't understand these wee enough to come up with good names.